external/source/exploits/CVE-2020-9850/payload/sbx/build-threadexec.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+git clone https://github.com/bazad/threadexec.git
+cd threadexec
```

Comment:
To make the build deterministic in the future can you please add a tag or commit checksum to check out? That way if something breaks up stream we'll still be able to reproduce the exploits that we can today.